src/SpectatorView.Native/SpectatorView.Compositor/dependencies.props

```diff
@@ -3,12 +3,10 @@
   <Import Project="dependencies.user.props" Condition="Exists('dependencies.user.props')"/>
   <ImportGroup Label="PropertySheets" />
   <PropertyGroup Label="UserMacros">
-    <!-- From: https://sourceforge.net/projects/opencvlibrary/files/opencv-win/3.2.0/opencv-3.2.0-vc14.exe/download -->
-    <OpenCV_vc14>$(SolutionDir)SpectatorView.Compositor\Dependencies\opencv-3.2.0\build\x64\vc14</OpenCV_vc14>
     <!-- From: https://www.blackmagicdesign.com/support -->
-    <DeckLink_inc>$(SolutionDir)SpectatorView.Compositor\Dependencies\BlackMagic DeckLink</DeckLink_inc>
+    <DeckLink_inc>x:\Blackmagic DeckLink SDK 10.11.4\Win\include</DeckLink_inc>
     <!-- From: https://github.com/elgatosf/gamecapture -->
-    <Elgato_Filter Condition="$(Elgato_Filter) == ''">$(SolutionDir)SpectatorView.Compositor\Dependencies\Elgato\gamecapture-master\VideoCaptureFilter</Elgato_Filter>
+    <Elgato_Filter Condition="$(Elgato_Filter) == ''">x:\elgatosf\gamecapture\VideoCaptureFilter</Elgato_Filter>
```

Comment:
><Elgato_Filter Condition="$(Elgato_Filter) == ''">x:\elgatosf\gamecapture\VideoCaptureFilter</Elgato_Filter> [](start = 4, length = 108)

Are these paths on your machine? or intended to be examples of how the paths work?

Reply:
We should likely have examples. Is SolutionDir typically contained in your user props file?

Reply:
SolutionDir is a property MSBuild defines. The previous directories are effectively where we had things checked in as part of Clarity Cam (and where I've been copying those dependencies into the project since then).

---
In reply to: [298390909](https://github.com/microsoft/MixedReality-SpectatorView/pull/50#discussion_r298390909) [](ancestors = 298390909)

Reply:
I'm going to remove the SolutionDir so that it doesn't look like these dependencies should be included in the repo.